// Exception Handling in c++

#include<iostream>
using namespace std;
int main(){
	
	int numerator, denominator;
	cout<<"Enter the numerator and denominator: "<<endl;
	cin>>numerator>>denominator;
	
	int result = numerator/denominator;
	cout<<"Division is: "<<result;
}
